src/content/blog/astro-2100.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: "Astro 2.10: Persistent State in View Transitions"
+description: "Persistent UI in View Transitions and our first look at Astro 3.0"
+publishDate: "August 3, 2023"
+authors:
+  - matthew
+coverImage: "/src/content/blog/_images/astro-2100/blog-hero-2_10-release.png"
+socialImage: "/src/content/blog/_images/astro-2100/og-image-2_10-release.png"
+lang: "en"
+---
+
+Astro 2.10 is out with improvements to our experimental View transitions support. Additionally we've released our first early beta of Astro 3.0. Read on to learn more!
+
+
+- **[Persistent UI in View transitions](#view-transitions-experimental)**
+- **[First Look at Astro 3.0](#astro-3)**
+
+If you already have Astro installed, you can upgrade it to 2.10 by running the upgrade command in your project (using your package manager of choice):
+
+```
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+While you're at it, upgrade any `@astrojs/*` integrations and adapters you have installed, too!
+
+# Persistent UI in View Transitions
+
+Our first release of the `<ViewTransition />` router allowed for smooth transitions between pages, but any stateful islands or elements you might have had would be rerendered.
+
+In 2.10 we have fixed this with a new `transition:persist` directive.
+
+With this directive, you can keep the state of DOM elements and islands on the old page when transitioning to the new page. [Here's a demo](https://stackblitz.com/edit/github-stfdbe-ikkp2e?file=src%2Fpages%2Findex.astro) of it in action.
```

Comment:
```suggestion
With this directive, you can keep the state of DOM elements and islands from the old page when transitioning to the new page. [Here's a demo](https://stackblitz.com/edit/github-stfdbe-ikkp2e?file=src%2Fpages%2Findex.astro) of it in action.
```
This *could* be read as "keeping stuff ON the last page"... to rephrase like retaining the stuff FROM the last page when you go TO the new page is just less ambiguous!